.gitignore

```diff
@@ -1 +1,3 @@
 .ipynb_checkpoints/
+contrib/secomea/*_ORIG.csv
+*.pdf
```

Comment:
Please do not ignore all .pdf files

As discussed this afternoon, in case of generated files I would advise to create them under i.e. `output/` subdirectory then ignore the directory instead